packages/service/src/domain/commands/processors/playback-current-command.ts

```diff
@@ -16,32 +16,36 @@ class PlaybackCurrentCommand extends CommandProcessor {
   }
 
   async execute(command: Command): Promise<CommandProcessingResponse> {
-    const includeQueue = command.rawArgs === 'queue';
+    const includeQueue = command.rawArgs?.includes('queue') ?? false;
+    const includePreview = command.rawArgs?.includes('embed') ?? false;
 
     const playerStateRequest: PlayerStateRequestEvent = { id: 'PlayerStateRequestEvent' };
     const currentPlayerState: PlayerState = await this.playerEventStream.sendToPrimaryPlayer(playerStateRequest);
 
     const text = this.buildMessage(
-      currentPlayerState.currentlyPlaying?.song ?? null,
+      currentPlayerState.currentlyPlaying ?? null,
       currentPlayerState.queue ?? [],
       includeQueue,
+      includePreview,
     );
 
     return new CommandProcessingResponseBuilder()
       .fromMarkdown(text)
       .build();
   }
 
-  private buildMessage(current: (Song | null), queue: Song[], includeQueue: Boolean): string {
-    const queueSongsTitles = includeQueue ? queue.map((song) => `  - ${song.name.truncated(50, false)}`) : [];
+  private buildMessage(current: (CurrentlyPlaying | null), queue: Song[], includeQueue: Boolean, includePreview: Boolean): string {
+    const queueSongsTitles = includeQueue ? queue.map((song) => `- ${song.name.truncated(50, false)}`) : [];
     const queueTitle = !queueSongsTitles.isEmpty() ? 'W kolejce:' : 'Playlista jest pusta';
-    const currentSongName = current?.name ?? '';
-    const currentYoutubeId = current?.youtubeId ?? '';
+    const currentSongName = current?.song.name ?? '';
+    let currentUrl = `https://www.youtube.com/watch?v=${current?.song.youtubeId}&t=${current?.time.toFixed(0)}`;
```

Comment:
It was requested during some music listening session. Eg when u have mix of many songs in one youtube video then you want timestamp to the particular one that is currently playing, maybe it could be introduced as yet another param?